README.md

```diff
@@ -31,35 +31,58 @@ The ```main``` branch of this repository contains samples configured for the lat
+
+> Note: The Toolkit Swift Package adds the ArcGIS Runtime SDK Swift Package as a dependency, so there is no need to add both separately.
+> 
+> New to Swift Package Manager? Visit [here](https://swift.org/package-manager/)
+
+## Building Samples Using Manually Installed SDK
+
 1. **Install** the ArcGIS Runtime SDK for iOS to a central location on your Mac as described [here](https://developers.arcgis.com/ios/get-started)
+1. **Fork** and then **clone** the repository
+1. **Download** the package repository for ArcGIS Runtime SDK for iOS [here](https://github.com/Esri/arcgis-runtime-ios), and extract it to `$HOME/arcgis-runtime-ios`
+1. **Create** a symbolic link to the ArcGIS Runtime SDK binary framework
+    > `ln -s $HOME/Library/SDKs/ArcGIS/Frameworks/ArcGIS.xcframework $HOME/arcgis-runtime-ios/ArcGIS.xcframework`
+1. **Open** the package manifest (Package.swift) for ArcGIS Runtime SDK
+    > Change target to
+    > `.binaryTarget(name: "ArcGIS", path: "ArcGIS.xcframework")`
+1. **Download** the package repository for ArcGIS Runtime Toolkit for iOS [here](https://github.com/Esri/arcgis-runtime-toolkit-ios), and extract it to the project directory
+1. **Drag** the Toolkit's folder into the Project navigator
+    > This action adds your dependency’s Swift package as a local package to your project.
+1. **Open** the Toolkit's package manifest
+    > Add `import FileProvider`
+    >
+    > Change dependency to
+    > `.package(name: "ArcGIS", path: "\(FileManager.default.homeDirectoryForCurrentUser.path)/arcgis-runtime-ios")`
 1. **Open** the `arcgis-ios-sdk-samples.xcodeproj` **project** file
 1. **Run** the `arcgis-ios-sdk-samples` app target
-    > If you get the error message saying _"This Copy Files build phase contains a reference to a missing file 'ArcGISToolkit.framework'"_, you probably didn't clone the repo to include it's submodule. See Step 1 above.
 
-## Building samples with CocoaPods
+> For more information on setting up local Swift Packages, visit [here](https://developer.apple.com/documentation/swift_packages/editing_a_package_dependency_as_a_local_package)
+
+## Building Samples Using CocoaPods
 
 1. **Fork** and then **clone** the repository
 1. **Install** the ArcGIS Runtime SDK for iOS by running the `pod install` command in the folder where you cloned this repository
 1. **Open** the `arcgis-ios-sdk-samples.xcworkspace` **workspace** file
-1. **Select** the `arcgis-ios-sdk-samples` project node, go to the `Build Phases` tab, and **delete** the phases for `Embed Frameworks`. (This phase conflicts with CocoaPods and is only required when using the installed SDK as described in the previous section)
-1. **Select** the `ArcGISToolkit.xcodeproj` project node and **delete** it. (this project dependency conflicts with CocoaPods and is only required when using the installed SDK as described in the previous section)
 1. **Run** the `arcgis-ios-sdk-samples` app target
```

Comment:
Does this still work if the swift packages are already configured in the project at the start?

Reply:
Good point. It seems there is a missing "Remove the toolkit from dependencies" step, but I have to test that out. 🤐

Reply:
Yes, I can test as well. And just adding the step of removing the toolkit package should be it.
You can remove here (you might already know):
![Screen Shot 2021-04-21 at 3 41 18 PM](https://user-images.githubusercontent.com/74069582/115629997-2a29d980-a2b8-11eb-8a46-cb5e5d86455b.png)

Reply:
Added a line of instruction.

Reply:
Yep removing the package and running `pod install` worked for me. Obviously had to change the pod to U10 and ignoring the build errors because it is missing U11 API.